CMakeLists.txt

```diff
@@ -14,6 +14,12 @@ endif ()
 
 if (YACLIB_CXX_STANDARD)
   set(CMAKE_CXX_STANDARD ${YACLIB_CXX_STANDARD})
+elseif (NOT CORO IN_LIST YACLIB_FLAGS)
+  message("Set default standard to c++17")
+  set(CMAKE_CXX_STANDARD 17)
```

Comment:
I am not sure that this is a good idea. @MBkkt WDYT ?
You can want not to use coro with C++20 standart, by default 
(coroutines may be not fully supported in some compilers)

Reply:
I think default standard should be small as possible

Maybe is good idea always set C++17, because some compilers support coroutine with C++17
And at the same time if compiler not supported user will be know about it and should be set standard manually

Reply:
@kononovk It's just a proposal. I really appreciate your comments.
These values will be set only if YACLIB_CXX_STANDARD hasn't been supplied. You always can specify any standard you want, as well as configure whether to use coros with this standard or not

Reply:
@MBkkt My idea is maybe better to have one default standard.
If user want coroutine, and not set standard, I think it is their problem.
And it can works, if not works user can specify more modern standard

Reply:
But in general current state also ok for me if it's necessary for conan